### PR TITLE
Readme change for powermax and powerstore namespace

### PR DESCRIPTION
**Pull request by arnavmohan0201**
# Description
This PR includes a small change in the readme file. For e2e testing of powermax and powerstore, namespace 'powermax' and 'powerstore' need to be created. This detail was missing from the previous prerequisites mentioned in the readme which has now been added. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1507 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by JacobGros — APPROVED** [review]

File: tests/README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Any time changes made to the operator are being checked into the main branch, sa
   - (if running sanity, powerflex, or modules suites) `test-vxflexos`
   - (if running sanity, powerscale, or modules suites) `isilon`
   - (if running unity suite) `unity`
+  - (if running powermax e2e tests) `powermax`
```
Comment (jooseppi-luna) [inline tests/README.md:46]: can we match the style of the above bullets -- e.g., (if running powermax suite) not (if running powermax e2e tests).

**Review by jooseppi-luna — CHANGES_REQUESTED** [review]

File: tests/README.md (hunk 1 above)
Comment (arnavmohan0201) [reply tests/README.md]: I have added these changes now. Thank you for the review!

File: tests/README.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ Any time changes made to the operator are being checked into the main branch, sa
   - (if running sanity, powerflex, or modules suites) `test-vxflexos`
   - (if running sanity, powerscale, or modules suites) `isilon`
   - (if running unity suite) `unity`
+  - (if running powermax suite) powermax
```
Comment (jooseppi-luna) [inline tests/README.md:46]: can we put the namespace in tildes to match above, e.g. `powerstore`

Comment (arnavmohan0201) [reply tests/README.md]: added!

**Review by jooseppi-luna — DISMISSED** [review]
looking good thanks!!

**Review by JacobGros — DISMISSED** [review]

**Review by JacobGros — APPROVED** [review]

**Review by JacobGros — CHANGES_REQUESTED** [review]
Do we need all 16 files changed?

**Pull request closed without merging**